packages/visualizations/src/components/Map/Combinations/NavigableMap.svelte

```diff
@@ -0,0 +1,85 @@
+<script lang="ts">
+    import type { BBox } from 'geojson';
+    import VectorChoropleth from '../WebGl/ChoroplethVectorTiles.svelte';
+    import type { ChoroplethDataValue, NavigableChoroplethOptions, NavigationMap } from '../types';
+
+    import BackButton from './BackButton.svelte';
+    import MiniMap from './MiniMap.svelte';
+
+    export let data: { value: ChoroplethDataValue[] };
+    export let options: NavigableChoroplethOptions;
+
+    $: ({ navigationMaps, colorScale, bbox, ...mainOptions } = options);
+
+    const defaultInteractive = true;
+    $: ({ interactive = defaultInteractive } = options);
+
+    let viewBox: BBox | undefined;
+    $: viewBox = bbox;
+
+    let active: number | undefined;
+    let height: number;
+
+    const setViewBox = (map: NavigationMap, i: number) => () => {
+        viewBox = map.bbox;
+        active = i;
+    };
+
+    const resetViewBox = () => {
+        active = undefined;
+        viewBox = bbox;
+    };
+
+    $: vectorOptions = { ...mainOptions, colorScale, viewBox };
+    /* Number of column x largest colmuns.
+     * The maps will then auto-adjust between 52 and 72 to fit the best
+     * It's stable and doesn't involves too much JS computaton either.
+     */
+    $: navMapsWidth = height ? Math.ceil((navigationMaps.length * 72) / height) * 72 : 72;
```

Comment:
In my tests (Firefox & Chrome), it seems the buttons are always at 72x72. Even when playing with the browser size etc. I'm not sure if something's missing.

As a result, since the buttons never take all their space, their block doesn't appear centered so it looks a bit weird.